docs/content/envoy-tutorial-istio.md

```diff
@@ -5,22 +5,23 @@ weight: 11
 ---
 
 [Istio](https://istio.io/latest/) is an open source service mesh for managing the different microservices that make
-up a cloud-native application. Istio provides a mechanism to customize the Envoy configuration generated by Istio Pilot
-using [EnvoyFilter](https://istio.io/latest/docs/reference/config/networking/envoy-filter/).
+up a cloud-native application. Istio provides a mechanism to use a service as an external authorizer with the 
+[AuthorizationPolicy API](https://istio.io/latest/docs/tasks/security/authorization/authz-custom/).
 
-This tutorial shows how Istio's EnvoyFilter can be configured to include Envoy's [External Authorization
-filter](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html) to delegate
-authorization decisions to OPA.
+This tutorial shows how Istio's AuthorizationPolicy can be configured to delegate authorization decisions to OPA.
 
 ## Prerequisites
 
 This tutorial requires Kubernetes 1.20 or later. To run the tutorial locally ensure you start a cluster with Kubernetes
 version 1.20+, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube) or
 [KIND](https://kind.sigs.k8s.io/).
 
-The tutorial also requires Istio v1.8.0 or later. It assumes you have Istio deployed on top of Kubernetes.
+The tutorial also requires Istio v1.19.0 or later. It assumes you have Istio deployed on top of Kubernetes.
 See Istio's [Quick Start](https://istio.io/docs/setup/kubernetes/install/kubernetes/) page to get started.
 
+If you are using an earlier version of Istio (1.9+), you will have to customize the `AuthorizationPolicy` in the to use the
```

Comment:
Nit: `in the ...` ?

Reply:
I guess we're referring to the `quick_start.yaml` file here.

Reply:
Fixed!